sbnobj/Common/Reco/MVAPID.h

```diff
@@ -0,0 +1,24 @@
+// Class for holding the result of an MVA PID
+
+#ifndef sbncode_MVAPID_H
```

Comment:
Capitalize include guards

Reply:
I was just following what has been done in other files: 
```FlashTriggerPrimitive.hh:#ifndef FlashTriggerPrimitive_SBN_hh_
MergedTrackInfo.hh:#ifndef sbncode_MergedTrack_HH
MVAPID.h:#ifndef sbncode_MVAPID_H
RangeP.h:#ifndef sbncode_rangep_HH
ScatterClosestApproach.h:#ifndef sbncode_scatterclosestapproach_H
ShowerSelectionVars.h:#ifndef sbncode_Showerselectionvars_H
SimpleFlashMatchVars.h:#ifndef sbncode_Simpleflashmatchvars_H
StoppingChi2Fit.h:#ifndef sbncode_stoppingchi2fit_H
Stub.h:#ifndef sbncode_Stub_HH
VertexHit.h:#ifndef sbncode_VertexHit_HH
```

I am happy to update either mine or all of them if this is something you feel strongly on?

Reply:
Maybe let's make another PR that changes all the include guards and renames the cc's to cxx's? The problem will come when it turns out that someone has been enforcing the opposite convention to me...

Reply:
Okay, that sounds good to me. I assume you have a strong opinion on `.h` vs `.hh` which are also used interchangeably currently and should be fixed in that PR

Reply:
`.h` please. Although changing those are presumably going to have an impact across all sorts of downstream repositories.

Reply:
Yeah, that will also require PRs in sbncode (and maybe sbndcode and icaruscode). If it sounds good to you, I will wait until this PR is merged then go through and update this?

Reply:
:+1: